client/src/entrypoints/admin/bulk-actions.js

```diff
@@ -16,6 +17,21 @@ const checkedState = {
   shouldShowAllInListingText: true,
 };
 
+/**
+ * Toggles the 'more' dropdown button in listing pages.
+ * @param {boolean} show - Determines if the button should be shown or not.
+ */
+function toggleMoreActionsDropdownBtn(show) {
+  const moreActionsDropdown = document.querySelector(
+    MORE_ACTIONS_DROPDOWN_BUTTON_SELECTOR,
+  );
+  if (moreActionsDropdown !== null) {
+    show === true
```

Comment:
this is where the linter is complaining. change it to an `if (show === true) { ... }`?

Reply:
Alright, I've changed it.